components/raftstore/src/store/util.rs

```diff
@@ -240,7 +240,7 @@ pub fn admin_cmd_epoch_lookup(admin_cmp_type: AdminCmdType) -> AdminCmdEpochStat
         AdminCmdType::CommitMerge => AdminCmdEpochState::new(true, true, true, false),
         AdminCmdType::RollbackMerge => AdminCmdEpochState::new(true, true, true, false),
         // Transfer leader
-        AdminCmdType::TransferLeader => AdminCmdEpochState::new(true, true, false, false),
+        AdminCmdType::TransferLeader => AdminCmdEpochState::new(false, false, false, false),
```

Comment:
Not recommended, ref:
https://github.com/tikv/tikv/blob/b0e2f87b19b75d0136cfa9c39412fba7a12ff87a/components/raftstore/src/store/util.rs#L218-L225